### PR TITLE
Add notes about desktop a11y on windows

### PR DESCRIPTION
**Pull request by VeselovAlex**

### CODE REVIEW

File: tutorials/Accessibility/Windows.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+## HiDPI issues
+### JDK support
+HiDPI support in Access Bridge was landed in [JDK-8279227](https://bugs.openjdk.java.net/browse/JDK-8279227). As for Feb/01/2022 this feature is not included in any released JDK, OpenJDK 17.0.4 release with this feature is planned for May 2022.
+
+### NVDA workaround
+NVDA 2021.3.1 does not handle widget position properly. Until they fix it we can override DPI awareness in NVDA compatibility settings as shown:
+
+![image](https://user-images.githubusercontent.com/3617329/151983695-2ff664a1-5ec5-4f27-9690-89037c6d9138.png)
```
Comment (olonho) [inline tutorials/Accessibility/Windows.md:17]: Guess we’d better keep image in git repo.

File: tutorials/Accessibility/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Accessibility support
```
Comment (olonho) [inline tutorials/Accessibility/README.md:1]: Could you please give general description of what we have now here.

**Comment by olonho** [comment]
Looks better, guess we could consider merging it.

File: tutorials/Accessibility/Windows.md (hunk 1 above)
Comment (StefanLobbenmeier) [reply tutorials/Accessibility/Windows.md]: I was wondering if the issue is still present in newer versions. I tried to reference the issues in nvda but did not find any references to compose multiplatform here: https://github.com/nvaccess/nvda/issues

**Pull request merged**